sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -163,6 +166,20 @@ CosmosKeyCredential cosmosKeyCredential() {
         return cosmosKeyCredential;
     }
 
+    /**
+     * Gets the boolean which indicates whether to only return the headers and status code in Cosmos DB response
+     * in case of Create, Update and Delete operations on CosmosItem.
+     *
+     * If true, this removes the resource from response. It reduces networking
+     * and CPU load by not sending the resource back over the network and serializing it
+     * on the client.
+     *
+     * @return a boolean indicating whether resource will be included in the response or not
+     */
+    boolean isReturnMinimalResponse() {
```

Comment:
.NET is using name 'NoResponseContent'.

/cc: @j82w

Reply:
NoContentResponseOnWrite was the current full name .NET using.

Reply:
Will go with `noContentResponseOnWrite` as the new name. Thanks.

Reply:
for similar configs we have "Enabled" as the prefix. To be consistent please add "Enabled" at the end. (`noContentResponseOnWriteEnabled`)